SegmentMesher/SegmentMesher.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Is this import needed?

Reply:
Without the import certain use cases will fail:
```log
>>> print("And this is?", file=sys.stderr)
  File "<console>", line 1
    print("And this is?", file=sys.stderr)
                              ^
SyntaxError: invalid syntax
```
```log
>>> from __future__ import print_function
>>> print("And this is?", file=sys.stderr)
And this is?
```
You will see that the import was included in all necessary files in the Slicer repo with this commit https://github.com/Slicer/Slicer/commit/08c6cd78fd25166773a35269ecd2f2c4fc2ea850